mailpoet/assets/js/src/newsletters/send/filter-segment.tsx

```diff
@@ -138,6 +148,33 @@ export function FilterSegment({
       </Tooltip>
       <div className="mailpoet-gap" />
       {filterSegmentSelect}
+      {isFilterSegmentEnabled && (
+        <p>
+          {ReactStringReplace(
+            __(
+              "Can't find the segment you're looking for? [link]Create new[/link]",
+              'mailpoet',
+            ),
+            /\[link\](.*?)\[\/link\]/g,
+            (match, i) => (
+              <a
+                className="mailpoet-link"
+                key={i}
+                rel="noopener noreferrer"
+                onClick={(event) => {
+                  event.preventDefault();
+                  context.saveDraftNewsletter(() => {
+                    window.location.href = `admin.php?page=mailpoet-segments#/new-segment?newsletterId=${item.id}`;
+                  });
+                }}
+                href={`admin.php?page=mailpoet-segments#/new-segment?newsletterId=${item.id}`}
```

Comment:
Details are important! Thank you for this comment. I extracted it to a variable in f6ac44a0c1690ec10d3d00d1daad330593baed31.

It wouldn't be strictly necessary to have the `href` since the click event should always get intercepted, but without the `href` the browser doesn't see it as an interactive element, so it doesn't exist in the tab order among other issues.